bin/console

```diff
@@ -14,7 +14,7 @@ set_time_limit(0);
 /**
  * @var Composer\Autoload\ClassLoader $loader
  */
-$loader = require __DIR__.'/autoload.php';
+$loader = require __DIR__.'/../app/autoload.php';
 
 $input = new ArgvInput();
 $env = $input->getParameterOption(array('--env', '-e'), getenv('SYMFONY_ENV') ?: 'dev');
```

Comment:
I think we should use a new array syntax. This converter can help you https://github.com/thomasbachem/php-short-array-syntax-converter